test/cvss.spec.js

```diff
@@ -15,6 +15,14 @@ describe("Score Tests", () => {
 
     const vector4 = CVSS("CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:H/A:N");
     expect(vector4.getScore()).toBe(8.2);
+  
+    //Temporal score tests
```

Comment:
I think it's a good idea to rather wrap the tests for temporal score into it's own `describe()` block.